database/migrations/2021_03_12_213148_create_wc_leads_table.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateWcLeadsTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('wc_leads', function (Blueprint $table) {
+            $table->id();
+            $table->string('email');
+            $table->string('first_name')->nullable();
+            $table->string('last_name')->nullable();
+            $table->string('position')->nullable();
+            $table->string('phone_number')->nullable();
+            $table->boolean('current_plan_under_cancellation')->nullable();
+            $table->datetime('current_plan_expires_at')->nullable();
```

Comment:
This will probably just be a date field instead of datetime.

Reply:
e2ca7f3c2241bbdbbd08ec12751f6399861353d0